<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: rgb(106, 110, 113);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="2">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: white;
    border-radius: 7px;
}</string>
          </property>
          <property name="text">
           <string> Apple</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="5">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: white;
    border-radius: 7px;
}</string>
          </property>
          <property name="text">
           <string> Tesla</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_19">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(160, 160, 160);
	color: rgb(255, 255, 255);
    border-radius: 7px;
}</string>
          </property>
          <property name="text">
           <string>Назва компанії</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: white;
    border-radius: 7px;
}</string>
          </property>
          <property name="text">
           <string> Amazon</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_20">
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(160, 160, 160);
	color: rgb(255, 255, 255);
    border-radius: 7px;
}</string>
          </property>
          <property name="text">
           <string>Ціна</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>Курс акцій</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="amazonLabel">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: rgb(255, 255, 255);
    /*border-radius: 10px;*/
}</string>
          </property>
          <property name="text">
           <string> </string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="appleLabel">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: rgb(255, 255, 255);
    /*border-radius: 10px;*/
}</string>
          </property>
          <property name="text">
           <string> </string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="teslaLabel">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: rgb(255, 255, 255);
    /*border-radius: 10px;*/
}</string>
          </property>
          <property name="text">
           <string> </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="Line" name="line_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="Line" name="line_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="4" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   background-color: rgb(135, 194, 50);
   border: 5px #000000;
   padding: 7px;
   border-radius: 50px;
border-image:  url(:money.ico);
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="weatherButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(135, 194, 50);
    border: 5px #000000;
    padding: 5px;
    border-radius: 50px;
	border-image:  url(:weather.ico);

}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="3" column="2">
         <widget class="QLabel" name="dollarLabel">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: rgb(255, 255, 255);
    /*border-radius: 10px;*/
}</string>
          </property>
          <property name="text">
           <string> </string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>Курс валют</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="euroLabel">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: rgb(255, 255, 255);
    /*border-radius: 10px;*/
}</string>
          </property>
          <property name="text">
           <string> </string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_14">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: white;
    border-radius: 7px;
}</string>
          </property>
          <property name="text">
           <string> Євро</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_12">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: white;
    border-radius: 7px;
}</string>
          </property>
          <property name="text">
           <string> Долар США</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_18">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(160, 160, 160);
	color: rgb(255, 255, 255);
    border-radius: 7px;
}</string>
          </property>
          <property name="text">
           <string>Назва валюти</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_16">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: white;
    border-radius: 7px;
}</string>
          </property>
          <property name="text">
           <string> Фунт стерлінгів</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_21">
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(160, 160, 160);
	color: rgb(255, 255, 255);
    border-radius: 7px;
}</string>
          </property>
          <property name="text">
           <string>Офіційний
курс</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="5">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="poundLabel">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: rgb(255, 255, 255);
    /*border-radius: 10px;*/
}</string>
          </property>
          <property name="text">
           <string> </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="Line" name="line_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="Line" name="line_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButton_4">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   background-color: rgb(135, 194, 50);
   border: 5px #000000;
   padding: 5px;
   border-radius: 50px;
   border-image:  url(:stocks.ico);
}

</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: white;
    border-radius: 7px;
}</string>
          </property>
          <property name="text">
           <string> Дата, час:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>Погода</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: white;
    border-radius: 7px;
}</string>
          </property>
          <property name="text">
           <string> Температура:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: white;
    border-radius: 7px;
}</string>
          </property>
          <property name="text">
           <string> Місто:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="tempLabel">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: rgb(255, 255, 255);
    /*border-radius: 10px;*/
}</string>
          </property>
          <property name="text">
           <string> </string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="dateTimeLabel">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: rgb(255, 255, 255);
    /*border-radius: 10px;*/
}</string>
          </property>
          <property name="text">
           <string> </string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(160, 160, 160);
	color: rgb(255, 255, 255);
    border-radius: 7px;
}</string>
          </property>
          <property name="text">
           <string>Вміст даних</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="cityLabel">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(97, 137, 48);
	color: rgb(255, 255, 255);
    /*border-radius: 10px;*/
}</string>
          </property>
          <property name="text">
           <string> </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="5">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="Line" name="line_10">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="Line" name="line_11">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="subsWeatherAction"/>
    <addaction name="subsStocksAction"/>
    <addaction name="subsValAction"/>
    <addaction name="separator"/>
    <addaction name="unsubsWeatherAction"/>
    <addaction name="unsubsStocksAction"/>
    <addaction name="undubsValAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="subsWeatherAction">
   <property name="text">
    <string>Підписатись на погоду</string>
   </property>
  </action>
  <action name="subsStocksAction">
   <property name="text">
    <string>Підписатись на курс акцій</string>
   </property>
  </action>
  <action name="subsValAction">
   <property name="text">
    <string>Підписатись на курс валют</string>
   </property>
  </action>
  <action name="unsubsWeatherAction">
   <property name="text">
    <string>Відписатись від погоди</string>
   </property>
  </action>
  <action name="unsubsStocksAction">
   <property name="text">
    <string>Відписатись від курсу акцій</string>
   </property>
  </action>
  <action name="undubsValAction">
   <property name="text">
    <string>Відписатись від курсу валют</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
